config/vufind/searches.ini

```
@@ -709,6 +709,11 @@ enabled = true
 default_handler = Solr
 ; Auto-submit autocomplete on click or enter
 auto_submit = true
+; Maximum number of suggestions shown. This will limit the number of items
+; in the drop down list created by the autocomplete javascript library. It
+; does not have any impact on the search for suggestions which is always
+; limited to 20. So setting a higher value than 20 has no effect.
```

This might be more accurate, and it's lighter on specifics so it will remain accurate if we change handler behavior in future.

```suggestion
; does not have any impact on the number of suggestions returned by
; the server-side autocomplete handler code. This number may vary from
; source to source.
```